allure-generator/src/main/javascript/components/node-search/NodeSearchView.js

```diff
@@ -0,0 +1,27 @@
+import './styles.scss';
+import {className, on} from '../../decorators';
+import template from './NodeSearchView.hbs';
+import {View} from 'backbone.marionette';
+
+@className('search')
+class NodeSearchView extends View {
+    template = template;
+
+    initialize({onSearch, searchQuery}) {
+        this.onSearch = onSearch;
+
+        this.searchQuery = searchQuery;
```

Comment:
You could pass a model here instead of `searchQuery/onSearch`.

@baev do you like it?

Reply:
by the way,  searchQuery is not needed to be passed, it left from the time I tried to re-rended this componentn on each state change